views/partials/head.html

```diff
@@ -7,7 +7,7 @@
   <meta name="description" content="{% if doc.description %}{{doc.description}} – {% endif %}Accelerated Mobile Pages Project">
```

Comment:
Also wrap the whole tag in `if doc.description`. Much better to have none, than a generic one.